imports/wasi_snapshot_preview1/args_test.go

```diff
@@ -158,8 +160,8 @@ func Test_argsSizesGet_Errors(t *testing.T) {
 			argvLen: validAddress,
 			expectedLog: `
 --> proxy.args_sizes_get(result.argc=65536,result.argv_len=0)
-	==> wasi_snapshot_preview1.args_sizes_get(result.argc=65536,result.argv_len=0)
-	<== EFAULT
+	--> wasi_snapshot_preview1.args_sizes_get(result.argc=65536,result.argv_len=0)
```

Comment:
interesting thing here is you can see the validation of memory doesn't hit the host function (`-->` not `==>`)